include/circt/Dialect/SMT/SMTIntOps.td

```diff
@@ -119,4 +119,13 @@ def IntCmpOp : SMTIntOp<"cmp", [Pure]> {
   let assemblyFormat = "$pred $lhs `,` $rhs attr-dict";
 }
 
+def Int2BVOp : SMTOp<"int2bv", [Pure]> {
+  let summary = "Convert an integer to an n-wide bitvector";
+  let arguments = (ins IntType:$input, ConfinedAttr<I32Attr, [IntNonNegative]>:$width);
```

Comment:
Why do we need a `width` attribute? The BV type already contains the width.

Reply:
Good point :woman_facepalming: thanks!